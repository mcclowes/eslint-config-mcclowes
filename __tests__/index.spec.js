var config = require('../index')

describe('eslint config', () => {
  it('exports correctly', () => {
    expect(config.rules).toMatchSnapshot()
  })
})