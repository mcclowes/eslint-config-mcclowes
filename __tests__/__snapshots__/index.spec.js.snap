// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`eslint config exports correctly 1`] = `
Object {
  "array-bracket-spacing": Array [
    "warn",
    "always",
  ],
  "comma-dangle": Array [
    "warn",
    "always",
  ],
  "comma-spacing": Array [
    "warn",
    Object {
      "after": true,
      "before": false,
    },
  ],
  "guard-for-in": "off",
  "lines-around-directive": Array [
    "warn",
    "always",
  ],
  "no-unused-expressions": Array [
    "warn",
    Object {
      "allowShortCircuit": true,
      "allowTaggedTemplates": true,
      "allowTernary": true,
    },
  ],
  "no-unused-vars": Array [
    "warn",
    Object {
      "argsIgnorePattern": "_+",
      "ignoreRestSiblings": true,
      "varsIgnorePattern": "_+",
    },
  ],
  "object-curly-spacing": Array [
    "warn",
    "always",
  ],
  "object-property-newline": Array [
    "warn",
    Object {
      "allowMultiplePropertiesPerLine": true,
    },
  ],
  "quotes": Array [
    "warn",
    "double",
  ],
  "react/boolean-prop-naming": Array [
    "error",
    Object {
      "propTypeNames": Array [
        "bool",
        "mutuallyExclusiveTrueProps",
      ],
    },
  ],
  "react/jsx-curly-spacing": Array [
    "warn",
    "always",
  ],
  "react/jsx-equals-spacing": Array [
    "warn",
    "always",
  ],
  "react/jsx-first-prop-new-line": Array [
    "warn",
    "multiline",
  ],
  "react/jsx-wrap-multilines": 1,
  "react/no-array-index-key": 1,
  "react/prop-types": 1,
  "react/self-closing-comp": Array [
    "error",
    Object {
      "component": true,
      "html": true,
    },
  ],
  "react/sort-prop-types": 1,
  "space-infix-ops": Array [
    "warn",
  ],
  "template-curly-spacing": Array [
    "warn",
    "always",
  ],
}
`;
